app/styles/_mobile-clients.less

```diff
@@ -6,4 +6,22 @@
       max-height: 350px;
     }
   }
+}
+
+.create-binding {
+  text-align: center;
+  .icon {
+    width: 64px;
+    height: 64px;
+    font-size: 4em;
+    margin: auto;
+    clear: both;
+  }
+  .note {
+    font-size: 1.3em;
+  }
+}
+
+.note {
```

Comment:
Can you namespace this? It's a bit general here. or should it be under `.create-binding` as well?